src/components/PostDetails/PostDetails.scss

```diff
@@ -7,6 +7,10 @@
     text-align: right;
   }
 
+  &__visiblelist {
```

Comment:
minor thing - but the name of the style is not ok

a) the list is "visible" but the display is 'none'
b) camelCase name is inconsistent with kebub-case used in other styles